ReactWindows/ReactNative.Net46.Tests/UIManager/DependencyObjectExtensionsTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿using NMock;
+using NUnit.Framework;
+using ReactNative.Bridge;
+using ReactNative.UIManager;
+using System;
+using System.Windows.Controls;
+
+namespace ReactNative.Tests.UIManager
+{
+    [TestFixture, RequiresSTA]
```

Comment:
>RequiresSTA [](start = 18, length = 11)

This smells of something being very wrong.  I've never seen a need for an attribute like this in unit tests...